sdk/cosmos/tests.live.yml

```diff
@@ -0,0 +1,429 @@
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - sdk/cosmos
+
+variables:
+  pomFile: './sdk/cosmos/pom.xml'
+  DefaultOptions: '--batch-mode -Dmaven.wagon.http.pool=false'
+  LoggingOptions: '-Dorg.slf4j.simpleLogger.defaultLogLevel=error -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
+
+jobs:
+  - job: 'Build'
+
+    pool:
+      vmImage: 'vs2017-win2016'
+
+    steps:
+    - task: Maven@3
+      displayName: 'Install reporting tools'
+      inputs:
+        mavenPomFile: ./eng/code-quality-reports/pom.xml
+        options: '$(DefaultOptions) -DskipTests -Dgpg.skip'
+        mavenOptions: '$(LoggingOptions)'
+        javaHomeOption: 'JDKVersion'
+        jdkVersionOption: '1.11'
```

Comment:
are we not using jdk8?

Reply:
Generally we build and test on JDK 8 and JDK 11 (although 11 will float to the latest LTS). In this case it is just the reporting tools, which we run under JDK 11 to get the strictest rules applied (e.g. for modules).